contracts/package.json

```diff
@@ -4,7 +4,7 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "compile:cairo": "hardhat starknet-compile",
+    "compile:cairo": "scarb --profile release build",
```

Comment:
I'd completely remove `compile:cairo` and invoke `scarb` where needed. No need to forward through yarn